public/createUser.js

```diff
@@ -15,7 +15,7 @@ let occupationSelection= $("#occupation");
 // let gameThree = $("gameThree");
 
 // Adding an event listener for when the form is submitted
-$(signUp).on("submit", function handleFormSubmit(event) {
```

Comment:
He said he wasnt sure if that was needed or not, we ended up taking it off and may have no affect.